spec/unit/plugin/pagination_spec.rb

```diff
@@ -53,6 +53,16 @@
     end
   end
 
+  describe '#total_pages' do
+    it 'returns the exact number of pages to accommodate all elements' do
+      users = rom.relation(:users).page(1).per_page(3)
+      expect(users.pager.total_pages).to eql(3)
+      
+      users = rom.relation(:users).per_page(9)
+      expect(users.pager.total_pages).to eql(1)
+    end
```

Comment:
they test the same but single it block should have only one expect block. You have describe or (context) to group them but here you should have two tests. One for every scenario you test.